packages/sdk-metrics/src/types.ts

```diff
@@ -23,3 +24,13 @@ export type CollectionOptions = CommonReaderOptions;
 export type ShutdownOptions = CommonReaderOptions;
 
 export type ForceFlushOptions = CommonReaderOptions;
+
+// NOTE: This is here because of the strange dependency on api?
```

Comment:
You can use a similar text that we used previously:

https://github.com/open-telemetry/opentelemetry-js/blob/fec6286127129ec0082e5dd080242501f15ea0af/packages/sdk-metrics/src/InstrumentDescriptor.ts#L45-L50